pkgs/data/icons/kuyen-icon-theme/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, stdenvNoCC, fetchFromGitLab, gtk3 }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "kuyen-icon-theme";
+  version = "747e0e01";
+
+  src = fetchFromGitLab {
+    owner = "froodo_alexis";
+    repo = "kuyen-icons";
+    rev = version;
+    sha256 = "sha256-6BK6+6CZhcPgDNuBIF9cYXGwS414NlL+KvRdsDzziaQ=";
+  };
+
+  nativeBuildInputs = [ gtk3 ];
+
+  dontDropIconThemeCache = true;
+
+  installPhase = ''
+    mkdir -p $out/share/icons/kuyen-icons
+    cp -r * $out/share/icons/kuyen-icons
+  '';
+
+  meta = with lib; {
+    description = "A colourful flat theme designed for Plasma desktop";
```

```suggestion
    description = "A colourful flat theme designed for Plasma Desktop";
```
would be nice to have a consistent capitalization